src/components/YourCompany/CompanyName/CorrectNameRequest.vue

```diff
@@ -122,6 +122,7 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
   }
 
   private validateEmailFormat (value: string): boolean {
+    // eslint-disable-next-line max-len
     const VALID_FORMAT = new RegExp(/^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/)
```

Comment:
Did this give lint errors? 
I've found regex does trigger the linter in some issues, but i've no idea why!

Reply:
Actually, no, it's didn't! I have no idea why.

I added the eslint flag here just to be explicit (and it case lint starts caring some time).